vignettes/Surveys_Walkthrough.Rmd

```diff
@@ -399,26 +398,230 @@ DT::datatable(head(study_data_raw, 25), rownames = FALSE, filter="top", options
 ```
 
 
-### Prep Kobo data
 
-In this chunk called `prep_kobo_data`, we run the command that creates `study_data_cleaned` as an exact copy of `study_data_raw`. This new data frame will be fully cleaned in subsequent sections.  
+
+### Prep data
+
+This dataset was produced with Kobo, but does not exactly align with the MSF data dictionary for mortality surveys. **Therefore, for this report we must treat it as "non-Kobo" and take actions to align it with the expected column names.**  
+
+Therefore, to prepare the data you should **comment/deactivate with #** the chunk called `prep_kobo_data`. Instead, we will use the approach outlined in the code chunk `prep_nonkobo_data`.  
+
+The main activity to prep the data is alignment of the column names to those expected by this R Markdown template.  
+
+
+First, we can un-comment and run the command that creates `study_data_cleaned` as an exact copy of `study_data_raw`. This new data frame will be fully cleaned in subsequent sections.  
 
 ```{r}
 study_data_cleaned <- study_data_raw
 ```
 
-The next uncommented command is unnecessary and should be deactivated with # symbols. We do not have a column called `violence_nature_other` in our survey dataset.  
+Next, we want to rename some of our columns. Using the command `msf_dict_rename_helper("mortality", varnames = "name")` we get a "helped" command copied to our clipboard. This contains many more variables than we need, but it is useful to review.  
+
+```{r, eval=FALSE, echo = TRUE}
```

Comment:
worth collapsing this code otherwise just painfully long

Reply:
fixed